.gitlab-ci.yml

```diff
@@ -569,7 +574,6 @@ cargo-check-macos:
         --substrate
         "$DEPENDENT_REPO"
         "$GITHUB_PR_TOKEN"
-    - cd "$DEPENDENT_REPO" && git rev-parse --abbrev-ref HEAD
```

Comment:
All the changes from this file are already in [master](https://github.com/paritytech/substrate/blob/master/.gitlab-ci.yml). Dirty rebase?

Reply:
I think it is because kiz-inner-hashing-runtime-migration is not merged with master (this PR is for this other PR).

Reply:
oh gotcha